code/datums/mood_events/needs_events.dm

```diff
@@ -3,6 +3,10 @@
 	description = "<span class='warning'><B>I'm so fat...</B></span>\n" //muh fatshaming
 	mood_change = -4
 
+/datum/mood_event/fathappy
+	description = "<span class='nicegreen'>I'm so fat...just how I should be!</span>\n"

```

Comment:
could probably just go with a nod to the original wellfed flavortext with less grammatical errors
```suggestion
	description = "<span class='nicegreen'>My belly feels round and full, just as it should be!</span>\n"
```

Reply:
Oh okay!  I figured it was more legible if it was broken up but i'll definitely make use of that in future.
Fair enough, I just went with the text that made the most sense to me at the time.
You aren't wrong about the foodie fetish...
And yeah, I just figured more commentary is better, i'll be less explicit if i do prs in the future.
Alright putnam, good to know :)

Reply:
> My belly feels round and full

Please don't spread your enthusiasm for vore around the sane portion of the codebase like that, deathride.